dicoogle/src/main/resources/webapp/js/components/plugin/pluginForm.jsx

```diff
@@ -1,8 +1,7 @@
 import React from "react";
 import * as PropTypes from "prop-types";
 import { ResultsSelected } from "../../stores/resultSelected";
-import dicoogleClient from "dicoogle-client";
-const Dicoogle = dicoogleClient();
+import Webcore from "dicoogle-webcore";
```

Comment:
:+1: Injecting methods into the global Dicoogle access object wasn't that good of an idea after all, so I'm OK with this.